Sources/Hilbert/spins.cc

```diff
@@ -142,4 +142,23 @@ void Spin::UpdateConf(Eigen::Ref<Eigen::VectorXd> v,
   }
 }
 
+void Spin::UpdateConfRowCol(Eigen::Ref<Eigen::VectorXd> vrow,
```

Comment:
What is the purpose of this function? sorry I got lost...

Reply:
it's equivalent to UpdateConf, but takes two visible configurations, `vrow` and `vrow`, and when `tochange[i] < NVisible()` it updates `vrow` otherwise it updates `vcol`.

Reply:
Resolved by removing this method